Deployment/templates/continous-deployment.yml

```diff
@@ -0,0 +1,28 @@
+parameters:
+  - name: ContinueEvenIfResourcesAreGettingDestroyed
+    type: boolean
+    default: false
+  - name: AzureSubscription
+    type: string
+
+steps:
+  - task: PowerShell@2
+    displayName: "terraform deployment"
+    inputs:
+      targetType: filePath
+      filePath: '$(Pipeline.Workspace)/terraformartifact/terraform_conditional_run.ps1'
+      arguments: '-deploymentResourceGroupName $(DeploymentResourceGroupName) -deploymentStorageAccountName $(DeploymentStorageAccountName) -workSpace $(Environment) -continueEvenIfResourcesAreGettingDestroyed $${{ parameters.ContinueEvenIfResourcesAreGettingDestroyed }}'
+    env:
+      ARM_CLIENT_ID: $(TERRAFORM-CLIENT-ID)
+      ARM_CLIENT_SECRET: $(TERRAFORM-CLIENT-SECRET)
+      ARM_TENANT_ID: $(TERRAFORM-TENANT-ID)
+      ARM_SUBSCRIPTION_ID: $(TERRAFORM-SUBSCRIPTION-ID)
+
+  - task: AzureCLI@2
+    displayName: "Website deployment"
+    inputs:
+      azureSubscription: "${{ parameters.AzureSubscription }}"
+      scriptType: 'bash'
+      scriptLocation: 'inlineScript'
+      inlineScript: |
+        az storage blob upload-batch -d '$web' --account-name $(website_storage_account_name) -s "$(Pipeline.Workspace)/AngularApp/ukho-fileshareservice-ui"
```

Comment:
This is good for adding/replacing files. I was thinking what happens when files are removed, it will leave old files as it is. Should we use `az storage blob delete-batch` to clean old files first and then upload new files?

Reply:
@vijendr13545 Yes Even I thought about it. But raise this PR and thought of adding this as part of other task as we need to give bit of thinking around it like do we need to keep backup or just straight forward delete will work.

Reply:
@ravi102465 good that there is a plan for it.